packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -94,13 +98,21 @@ export const RichTextBlock = memo(function RichTextBlock({
           <EmojiPicker doWithEmoji={emoji => editor.insertText(emoji)} />
         </EditorSubMenuButton>
       </Toolbar>
+      {isEmpty && ( // Alternative placeholder
+        <div onClick={() => ReactEditor.focus(editor)} style={{color: '#cad5e4'}}>
+          {t('blocks.richText.startWriting')}
+        </div>
+      )}
       <Editable
         readOnly={disabled}
-        placeholder={t('blocks.richText.startWriting')}
+        // placeholder={t('blocks.richText.startWriting')}  # causes focusing problems on firefox !
         renderElement={renderElement}
         renderLeaf={renderLeaf}
         onBlur={() => {
           setLocation(editor.selection)
+          setIsEmpty(
+            JSON.stringify(Array.from(editor.children)) === JSON.stringify(createDefaultValue())
```

Comment:
maybe move this to a function inside the editor? `editor.isEmpty()` which returns a boolean. Might be useful in other places.

Reply:
Good idea. Will update a suggestion according to https://docs.slatejs.org/concepts/07-plugins#helper-functions .